nautobot/circuits/templates/circuits/inc/circuit_termination.html

```diff
@@ -42,7 +42,7 @@
                 </td>
             </tr>
             <tr>
-                <td>Termination</td>
```

Comment:
Weren't you planning to go with hiding this row altogether if it's not terminated to a site?

Reply:
Already handled at line 34